docker-bake.hcl

```diff
@@ -33,6 +33,14 @@ function base-packages {
   result = (
     equal(distro(item), "ubuntu") || equal(distro(item), "nvidia/cuda") || equal(distro(item), "intel/oneapi") ?
       join(" ", [
+        "autoconf",
```

Comment:
These should be passed as `extra_packages` in the LDMS image, we don't want to add unnecessary dependencies to all images.